datamanager/splitLargeZip/splitZip.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+"""
+This script is used to split zip files (DM exports) that are above 1GB in size.
+
+Usage:
+    splitZip.py --path C:\\path\\to\\zip
+
+The result of running the script is multiple zip files that are all below 1GB.
+"""
+import argparse
+import csv
+import os
+import tempfile
+import zipfile
+from dataclasses import dataclass
+from io import TextIOWrapper
+from pathlib import Path
+
+ONEGB = 1000000000
+
+
+@dataclass
+class configs:
+    path_to_schema = ""
+    path_to_split = ""
+    start_size = 0  # size of schema and split file
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--path", help="the path of the zip")
+    args = parser.parse_args()
+    processZip(args.path)
+
+
+def processZip(path):
+    with tempfile.TemporaryDirectory() as temp_zip_contents:
+        with zipfile.ZipFile(path, "r") as zip_ref:
+            zip_ref.extractall(temp_zip_contents)
+            dictionary_paths = {
+                path.name: path for path in list(Path(temp_zip_contents + "/" + path.rstrip(".zip")).iterdir())
```

Comment:
```suggestion
                path.name: path for path in Path(temp_zip_contents) / path.stem).iterdir()
```
you don't need list conversion as `for ... in` knows to user generators and iterators. Also if `path` would be of type Path you can use `.stem` instead os rstrip which is more hackish. Path joining can be done even as in the suggestion with `/` or with `.joinpath()`.
Please check https://docs.python.org/3/library/pathlib.html - it also has alternatives from using strings to using Path objects. Also this for iterators and generators: https://anandology.com/python-practice-book/iterators.html